actors/builtin/reward/reward_logic.go

```diff
@@ -5,8 +5,30 @@ import (
 	big "github.com/filecoin-project/specs-actors/actors/abi/big"
 )
 
-var BaselinePowerAt = func(epoch abi.ChainEpoch) abi.StoragePower {
-	return big.NewInt(1 << 40) // PARAM_FINISH
+const BaselineExponentString = "340282663082994238536867392845056089438"
+
+// Baseline function = BaselineInitialValue * (BaselineExponent) ^(t), t in epochs
+var BaselineExponent big.Int     // Q.128
+var BaselineInitialValue big.Int // Q.0
+
+func init() {
+	BaselineExponent = big.MustFromString(BaselineExponentString)
+	BaselineInitialValue = big.Lsh(big.NewInt(1), 60) // 1 EiB
+}
+
+// Initialize baseline power for epoch -1 so that baseline power at epoch 0 is
+// BaselineInitialValue.
+func InitBaselinePower() abi.StoragePower {
```

Comment:
@anorth Perhaps it would be clearer if we special cased epoch 0 in `updateToNextEpoch`?  We'd also avoid the 1 byte difference between `BaselineInitialValue` and the constructor set `st.ThisEpochBaselinePower`.

Reply:
Your call